### PR TITLE
Feature/conformance alerts

### PR DESCRIPTION
**Pull request by KarenAJ**
Added conformance alerts functionality in XMS.

### CODE REVIEW

File: app/core/management/utils/xds_internal.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import logging
```
Comment (jdiouf) [inline app/core/management/utils/xds_internal.py]: File name says XDS

**Review by jdiouf — APPROVED** [review]

File: app/core/models.py [hunk 2]
```diff
@@ -24,3 +26,39 @@ def save(self, *args, **kwargs):
...
+    """Model for Email Configuration """
+
+    sender_email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter sender email address to send log data from',
+        default='openlxphost@gmail.com')
+
+    def save(self, *args, **kwargs):
+        if not self.pk and SenderEmailConfiguration.objects.exists():
+            raise ValidationError('There is can be only one '
```
Comment (mattdrayer) [inline app/core/models.py]: Nit: Grammar

File: app/core/models.py [hunk 3]
```diff
@@ -24,3 +26,39 @@ def save(self, *args, **kwargs):
...
+        return super(ReceiverEmailConfiguration, self).save(*args, **kwargs)
+
+
+class SenderEmailConfiguration(models.Model):
+    """Model for Email Configuration """
+
+    sender_email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter sender email address to send log data from',
+        default='openlxphost@gmail.com')
```
Comment (mattdrayer) [inline app/core/models.py]: We should probably refrain from using a Gmail address here, even if we think it's not a real one -- you just never know.

**Review by mattdrayer — APPROVED** [review]
Just a couple small things, otherwise LGTM!

**Pull request merged**